.github/workflows/pipeline_release.yml

```diff
@@ -7,106 +7,119 @@ on:
+        with:
+          name: packages
+          path: |
+            nuxeo-ai-core-package/target/nuxeo-ai-core-$RELEASED_VERSION.zip
 
-    - name: Archive packages
-      uses: actions/upload-artifact@v2
-      with:
-        name: packages
-        path: |
-          nuxeo-ai-core-package/target/nuxeo-ai-core-$RELEASED_VERSION.zip
+      - name: Publish Nuxeo packages
+        env:
+          CONNECT_PREPROD_URL: https://nos-preprod-connect.nuxeocloud.com/nuxeo
+        run: |
+          PACKAGE="nuxeo-ai-core-package/target/nuxeo-ai-core-$RELEASED_VERSION.zip"
+          STATUS_CODE=`curl -i --silent --output publish-req.output -w "%{http_code}" -u "${{ secrets.CONNECT_PREPROD_AUTH }}" -F package=@$PACKAGE "$CONNECT_PREPROD_URL/site/marketplace/upload?batch=true"`
+          cat publish-req.output
+          if [[ "$STATUS_CODE" != "200" ]]
+          then
+            exit 1
+          else
+            exit 0
+          fi
 
-    - name: Publish Nuxeo packages
-      env:
-        CONNECT_PREPROD_URL: https://nos-preprod-connect.nuxeocloud.com/nuxeo
-      run: |
-        PACKAGE="nuxeo-ai-core-package/target/nuxeo-ai-core-$RELEASED_VERSION.zip"
-        STATUS_CODE=`curl -i --silent --output publish-req.output -w "%{http_code}" -u "${{ secrets.CONNECT_PREPROD_AUTH }}" -F package=@$PACKAGE "$CONNECT_PREPROD_URL/site/marketplace/upload?batch=true"`
-        cat publish-req.output
-        if [[ "$STATUS_CODE" != "200" ]]
-        then
-          exit 1
-        else
-          exit 0
-        fi
+      - name: Bump release version
+        id: bump_version
+        uses: christian-draeger/increment-semantic-version@1.0.2
+        with:
+          current-version: ${{ github.event.inputs.version }}
+          version-fragment: ${{ github.event.inputs.increment }}
 
-    - name: Bump release version
-      id: bump_version
-      uses: christian-draeger/increment-semantic-version@1.0.2
-      with:
-        current-version: ${{ github.event.inputs.version }}
-        version-fragment: ${{ github.event.inputs.increment }}
+      - name: Post Release (in master or master-10.10)
+        run: |
+          git checkout ${{ github.base_ref }}
+          find ./ -type f -exec sed -i '' -e "s/$CURRENT_VERSION/${{ steps.bump_version.outputs.next-version }}/" {} \;
+          git clean -f -d
+          git add .
+          git commit -m "Post Release $CURRENT_VERSION"
+          git push origin ${{ github.base_ref }}
 
-    - name: Post Release (in master or master-10.10)
-      run: |
-        git checkout ${{ github.base_ref }}
-        find ./ -type f -exec sed -i '' -e "s/$CURRENT_VERSION/${{ steps.bump_version.outputs.next-version }}/" {} \;
-        git clean -f -d
-        git add .
-        git commit -m "Post Release $CURRENT_VERSION"
-        git push origin ${{ github.base_ref }}
+  notify-on-error:
+    runs-on: ubuntu-latest
+    needs: release-publish
+    if: ${{ failure() && github.event_name == 'push' }}
```

Comment:
This workflow is triggered by a manual `workflow_dispatch`

https://github.com/nuxeo/nuxeo-ai/blob/63930dbee31bd633c43c4b1eca91802619888bca/.github/workflows/pipeline_release.yml#L4

I think the event name wouldn't be push. Can you remove the `github.event_name == 'push'` condition? Since this is always manually run, we can alert on all workflow failures for this workflow.